src/command_modules/azure-cli-container/azure/cli/command_modules/container/_validators.py

```diff
@@ -37,6 +42,13 @@ def validate_gitrepo_directory(ns):
         raise CLIError("The git repo directory cannot contain '..'")
 
 
+def validate_image(ns):
+    if ns.image.split(':')[0] in short_running_images and not ns.command_line:
+        logger.warning('Image "%s" has no long running process. '
+                       'A command is required for the container to stay running.',
+                       ns.image)
```

Comment:
What does this mean?  To me it sounds like some creates are LRO and some aren't... is that the case?  What command is "required for the container to stay running"?